### PR TITLE
Update changelog for v2.13.0

### PR DESCRIPTION
**Pull request by florayan**

### CODE REVIEW

**Comment by TAOXUY** [comment]
Please use `Update changelog for vXXX` as title

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# Release 2.13.0 08-07-2020
+
+- Update envoy to 7/6/2020 (#217)
```
Comment (TAOXUY) [inline CHANGELOG.md:3]: Only keep customer facing bugs/feature requests

- Create FilterStats in client_cache (#216)
- Fix bug in processing HTTP response, unit tests for client cache CHECK (#212)
- Support api config versioning:  add v6 to api folder name and package name (#210)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+- Create FilterStats in client_cache (#216)
+- Fix bug in processing HTTP response, unit tests for client cache CHECK (#212)
+- Add api/README.md and api/script/update_version.h (#211)
+- Support api config versioning:  add v6 to api folder name and package name (#210)
```
Comment (qiwzhang) [inline CHANGELOG.md:5]: remove line 4 and 6.

Comment (qiwzhang) [reply CHANGELOG.md:5]: remove line 6

Comment (florayan) [reply CHANGELOG.md:4]: Done. Thanks!

Comment (qiwzhang) [reply CHANGELOG.md:4]: Please add back "Update Envoy .." line.  It will be useful to users too

Comment (florayan) [reply CHANGELOG.md]: Done, Thanks!

**Comment by TAOXUY** [comment]
> Please use `Update changelog for vXXX` as title

Update title?

**Comment by florayan** [comment]
/assign  qiwzhang@

> > Please use `Update changelog for vXXX` as title
> 
> Update title?

Done. Thanks:)

**Pull request merged**